cpp/arcticdb/stream/python_bindings.cpp

```diff
@@ -24,6 +24,12 @@ namespace py = pybind11;
 namespace arcticdb {
 using namespace arcticdb::python_util;
 
+std::vector<FieldRef> field_collection_to_ref_vector(const FieldCollection& fields){
+    auto result = std::vector<FieldRef>(fields.size());
```

Comment:
This will call the default constructor of `FieldRef` `fields.size()` times, `reserve` avoids this